src/swagger.yml

```diff
@@ -2006,6 +2006,265 @@ paths:
                           locale: "it"
                           value: "Attualmente non disponibile"
                     shop_url: null
+  # Appraisals
+  /appraisals:
+    get:
+      summary: List wine appraisals
+      description: >
+        Use this operation to get a list of all wine appraisals (Weinbewertungen).
+      tags:
+        - Appraisals
+      operationId: listAppraisals
+      security:
+        - appKey: []
+#        - appId: []
+      parameters:
+        - name: f.email
+          description: >
+            Filter by email, e.g. "f.email=__in_testuser1@denner.ch,testuser1@denner.ch" will return all appraisals which have been created by testuser1@denner.ch or testuser1@denner.ch.
+          in: query
+          type: string
+        - $ref: '#/parameters/PageParam'
+        - $ref: '#/parameters/PageSizeParam'
+        - $ref: '#/parameters/SortParam'
+      responses:
+        200:
+          description: Successful listing
+          schema:
+            $ref: '#/definitions/AppraisalsResponse'
+          examples:
+            application/json:
+
+              appraisals:
+                -
+                  id: "a8c82c69-e975-4d3b-b0ba-aae6ed961cd7"
+                  created_on: "2015-10-20T14:37:56+0200"
+                  title: "Ein Primitivo passt immer."
+                  rating: 4
+                  language: "de"
+                  article:
+                    id: "050202"
+                  marked: true
+                  processed: false
+                  approved: false
+                  approved_image: false
+                  type: "wine_selfie"
```

Comment:
The example is what the name says: An example :-)
Thus id doesn't need to contain optional fields (but it can, of course).

The spec on the other hand must define all fields.